### PR TITLE
Add muted badge colour

### PR DESCRIPTION
**Pull request by stredger**
## Screenshots
Screen Shot 2020-02-27 at 11 09 50 AM

### CODE REVIEW

File: packages/riipen-ui-docs/src/pages/components/badge/badge.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The badge auto hides with `content` is zero. You can override this with the show
 
 ## Maximum value
 
- You can usr the `max` property to cap the value of the badge content.
+You can usr the `max` property to cap the value of the badge content.
```
Comment (bolinkd) [inline packages/riipen-ui-docs/src/pages/components/badge/badge.md]: use*

File: packages/riipen-ui/src/components/Badge.jsx [hunk 2]
```diff
@@ -172,6 +178,10 @@ class Badge extends React.Component {
             background-color: ${theme.palette.negative.main};
             color: ${theme.palette.negative.contrast};
           }
+          .muted {
+            background-color: ${theme.palette.grey[400]};
+            color: ${theme.palette.negative.contrast};
+          }
```
Comment (bolinkd) [inline packages/riipen-ui/src/components/Badge.jsx]: I am going to assume these colours are confirmed somewhere as I dont know where to look 🙈

Comment (stredger) [reply packages/riipen-ui/src/components/Badge.jsx]: `riipen-ui/src/themes/*.js` for all your colouring needs!

Comment (bolinkd) [reply packages/riipen-ui/src/components/Badge.jsx]: I meant ui designs for the muted badge

**Pull request merged**